create.ps1

```diff
@@ -1,258 +1,449 @@
-###################################################################
-# HelloID-Conn-Prov-Target-KPNLisa-Create
+#################################################
+# HelloID-Conn-Prov-Target-KPN-Lisa-Create
+# Create and update or correlate to account
 # PowerShell V2
-###################################################################
+#################################################
```

Comment:
In het algemeen zou ik willen voorstellen om alle comments in de code te verwijderen. De meeste comments voegen niet veel toe omdat de code opzich zelf beschrijvend genoeg is. De regions kunnen mogelijk op een hoger niveau worden geplaatst. Bv. een region om het account te creëren / bij te werken.

Reply:
Deze zijn juist weer in overleg toegevoegd, met name voor andere consultants/support dat ze direct weten welke versie van het script er staat en een korte samenvatting van wat het script doet, zodat ze niet het hele script hoeven door te leven.
Soms is een create een correlate-only, soms een create of correlate en in dit geval create + update of correlate.